README.md

```diff
@@ -77,6 +77,8 @@ To start a deployment in the cloud:
 $ spatial alpha cloud launch -d <your-deployment-name> -a <your-assembly-name>
 ```
 
+As above, you can run these commands from the /SpatialOS directory or use the `--main_config=\<path to spatialos.json\>` flag to point to your project config when running the commands from another directory.
```

Comment:
Nit: formatting:
```suggestion
As above, you can run these commands from the `SpatialOS` directory or use the `--main_config=<path to your spatialos.json>` flag to point to your project configuration file when running the commands from another directory.
```

Reply:
Fixed (and applied the nit to the line I copied from the local launch section.)